jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/oss/OssPosition3D.java

```diff
@@ -0,0 +1,29 @@
+package us.dot.its.jpo.ode.plugin.j2735.oss;
+
+/**
+ * Created by anthonychen on 3/3/17.
+ */
+
+import us.dot.its.jpo.ode.j2735.dsrc.Elevation;
+import us.dot.its.jpo.ode.j2735.dsrc.Latitude;
+import us.dot.its.jpo.ode.j2735.dsrc.Longitude;
+import us.dot.its.jpo.ode.j2735.dsrc.Position3D;
+import us.dot.its.jpo.ode.plugin.j2735.J2735Position3D;
+
+public class OssPosition3D {
```

Comment:
We have already defined OssPosition3D class in ode-147. We need to merge the two branches.